gcore/gdalopeninfo.cpp

```diff
@@ -404,7 +404,12 @@ GDALOpenInfo::~GDALOpenInfo()
 /************************************************************************/
 
 /** Return sibling files.
- * @return sibling files. Ownership below to the object.
+ *
+ * If the list of sibling files has not already been established, it will be,
+ * unless the GDAL_DISABLE_READDIR_ON_OPEN configuration option has been set to
+ * YES or EMPTY_DIR when this instance was constructed.
+ *
+ * @return sibling files. Ownership belong to the object.
```

Comment:
To which object? The caller or `this` (I assume the second but maybe clearer to mention ownership wasn't "transfered") ?

Reply:
to "this". Comment clarified to reflect ... this :-)